src/goto-checker/README.md

```diff
@@ -30,10 +30,15 @@ verification algorithm. Incremental goto checkers keep the state of the
 analysis and can be queried by the goto verifier repeatedly to return
 their results incrementally. A query to an incremental goto checker
 either returns when a violated property has been found or the
-verification algorithm has terminated. If a violated property has been
-found then additional information (e.g. a trace) can be retrieved
-from the incremental goto checker (if it supports that).
-See \ref incremental_goto_checkert.
+verification algorithm has terminated.  See \ref incremental_goto_checkert.
+Incremental goto checkers can provide additional functionality by implementing
+the respective interfaces:
+* \ref goto_trace_providert: If a violated property has been
+  found then a trace can be retrieved from the incremental goto checker.
+* \ref fault_localization_providert: If a violated property has been
+  found then a likely fault location can be determined.
+* \ref witness_providert: A correctness witness can be output (for violated
```

Comment:
None of these `\ref` appear to resolve according to CI.

Reply:
Also: "correctness witness" and "violated property" don't go together.

Reply:
Same trap again: there must be a space to separate the colon from the ref.